docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

```diff
@@ -100,15 +98,16 @@ Here is an example that shows how to exclude the label `Team`. You can choose be
 
 An alert's label set can contain three types of labels:
 
-- Labels from the datasource,
-- Custom labels specified in the alert rule,
-- A series of reserved labels, such as `alertname` or `grafana_folder`.
+- The data source query. For example, if you were monitoring temperature readings and each time series for these readings would have a sensor_id, and a location label. Those labels are used to provide additional context or dimensions to the metric data, helping to differentiate between different time series.
 
-### Custom Labels
+- Some are automatically added by Grafana (i.e. alertname and grafana_folder). These are Grafana reserved labels.
 
-Custom labels are additional labels configured manually in the alert rule.
+- Others you can define yourself to help filter data in your alert rules. They act as a unique identifier of an alert. In addition, labels allow searching, silencing, or routing alerts. 
+And, you can also template them. For example in your alert, you could add a label that uses templating to create more dynamic and customizable alerting. E.g. `environment` `=` `{{ your text/template }}`.
 
-Ensure the label set for an alert does not have two or more labels with the same name. If a custom label has the same name as a label from the datasource then it will replace that label. However, should a custom label have the same name as a reserved label then the custom label will be omitted from the alert.
+{{<admonition type="note">}}
+Ensure the label set for an alert does not have two or more labels with the same name. If a label has the same name as a label from the datasource then it will replace that label. However, should a label have the same name as a reserved label then the label will be omitted from the alert.
```

Comment:
```suggestion
Ensure the label set for an alert does not have two or more labels with the same name. If a label has the same name as a label from the data source then it will replace that label. However, should a label have the same name as a reserved label then the label will be omitted from the alert.
```